packages/react/src/React.js

```diff
@@ -96,6 +97,7 @@ export {
   REACT_FRAGMENT_TYPE as Fragment,
   REACT_PROFILER_TYPE as Profiler,
   REACT_STRICT_MODE_TYPE as StrictMode,
+  REACT_DEBUG_TRACE_MODE_TYPE as DebugTraceMode,
```

Comment:
Can this export be `undefined` outside of DEV and when the flag is off?

Reply:
Yeah I can make it conditional if the feature flag is disabled.